### PR TITLE
Refs [TASK][34088] Create UI Platform

### PR DESCRIPTION
**Pull request by Hoang-Kelvin**
## Related Tickets
- [#Task 34088: Create UI Platform](https://edu-redmine.sun-asterisk.vn/issues/34088)
## WHAT
- Create TextView Title,Image View,..
## Evidence (Screenshot or Video)
image

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/java/com/project/gamedb/ui/platform/PlatformContract.kt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package com.project.gamedb.ui.platform
+
+interface PlatformContract {
+}
```
Comment (thonc-2135) [inline app/src/main/java/com/project/gamedb/ui/platform/PlatformContract.kt]: có j đó sai sai

File: app/src/main/res/layout/fragment_platform.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/image_background"
+    android:paddingBottom="@dimen/dp_70"
+    tools:context=".ui.platform.PlatformFragment">
+
+    <ImageView
+        android:id="@+id/ImagePcPlatform"
```
Comment (thonc-2135) [inline app/src/main/res/layout/fragment_platform.xml]:
```suggestion
        android:id="@+id/imagePcPlatform"
```
cả mấy cái dưới nữa nhé

File: app/src/main/java/com/project/gamedb/ui/platform/PlatformPresenter.kt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package com.project.gamedb.ui.platform
+
+class PlatformPresenter {
+}
```
Comment (thonc-2135) [inline app/src/main/java/com/project/gamedb/ui/platform/PlatformPresenter.kt]: tương tự

File: app/src/main/java/com/project/gamedb/ui/platform/PlatformFragment.kt [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        get() = R.layout.fragment_platform
+
+    override fun initData() {
+
+    }
+
+    override fun initAdapter() {
+
+    }
+
```
Comment (huytq-2422) [inline app/src/main/java/com/project/gamedb/ui/platform/PlatformFragment.kt:17]:
```suggestion
    }
```

File: app/src/main/java/com/project/gamedb/ui/platform/PlatformFragment.kt [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+}
```
Comment (huytq-2422) [inline app/src/main/java/com/project/gamedb/ui/platform/PlatformFragment.kt]:
```suggestion
    }
}
```

File: app/src/main/java/com/project/gamedb/ui/main/MainActivity.kt [hunk 6]
```diff
@@ -9,12 +9,14 @@ import com.google.android.material.bottomnavigation.BottomNavigationView
...
 import com.project.gamedb.base.BaseActivity
+import com.project.gamedb.ui.platform.PlatformFragment
 import com.project.gamedb.ui.popular.PopularFragment
 import kotlinx.android.synthetic.main.activity_main.*
 
 class MainActivity : BaseActivity() {
     private val popularFragment = PopularFragment()
-    private val mainAdapter : MainAdapter = MainAdapter(supportFragmentManager)
+    private val platformFragment = PlatformFragment()
+    private val mainAdapter: MainAdapter = MainAdapter(supportFragmentManager)
```
Comment (huytq-2422) [inline app/src/main/java/com/project/gamedb/ui/main/MainActivity.kt:19]:
```suggestion
    private val mainAdapter = MainAdapter(supportFragmentManager)
```

**Pull request merged**